src/main/java/hello/until/item/controller/ItemController.java

```diff
@@ -22,4 +22,16 @@ public ReadItemResponse readItem(@PathVariable long id) {
                 .map(ReadItemResponse::new)
                 .orElseThrow(() -> new ResponseStatusException(HttpStatus.NOT_FOUND));
     }
+
+    @PostMapping("")
```

Comment:
```suggestion
@PostMapping
```

그냥 이렇게 써도 됐었던 것 같아요~

Reply:
그렇군요 수정하겠습니다 :)